### ISSUE TITLE
Multiple date fields not working

### ISSUE THREAD

**Issue by itmunky** [ivandokov/phockup#113]
When using the date field option -f | --date-field, i can get a single date field to work however when using multiple fields as explained 
   "To set multiple fields to be tried in order until a valid date is found, just join them with spaces in a quoted string like "CreateDate FileModifyDate".
it just copies the photos into the unknown folder. It does not take into account either of the specified date fields.

```bash
itmunky@pop-os:/mnt/unraid/phockup$ ls -lah | head
-rw-rw-rw- 1 99 users  4.0M Oct 23  2018 20160827_203818.jpg

itmunky@pop-os:/mnt/unraid/phockup$ exiftool -time:all -mimetype -j 20160827_203818.jpg 
[{
  "SourceFile": "20160827_203818.jpg",
  "FileModifyDate": "2018:10:23 08:39:28+08:00",
  "CreateDate": "2016:08:27 20:38:17",
  "MIMEType": "image/jpeg"
}]

itmunky@pop-os:/mnt/unraid$ phockup -y -d "YYYY/YYYY-MM - M YY'" -f="CreateDate FileModifyDate" phockup phockupsorted
phockup/20160827_203818.jpg => phockupsorted/unknown/20160827_203818.jpg
```

**Comment by ivandokov** [comment]
Fixed in version 1.6.1

**Comment by itmunky** [comment]
> Fixed in version 1.6.1

Legend. Thankyou.
